src/components/NcBreadcrumb/NcBreadcrumb.vue

```diff
@@ -44,7 +43,7 @@ Renders a button element when given no redirection props, otherwise, renders <a/
 			:menu-name="name"
 			:title="title"
 			:force-name="true"
-			:container="`.vue-crumb[${crumbId}]`"
+			:container="$refs.crumb"
```

Comment:
But at the moment of rendering `$refs` is empty...

Reply:
Too expensive to re-render the `NcActions` in the second iteration when `refs` is populated?

Otherwise can just use a proper attribute for this, but the current state can never really work.